build-system/tasks/bundle-size.js

```diff
@@ -99,53 +114,64 @@ function storeBundleSize() {
     return;
   }
 
-  const bundleSize = `${getGzippedBundleSize()}KB`;
-  const commitHash = gitCommitHash();
-  const githubApiCallOptions = Object.assign(buildArtifactsRepoOptions, {
-    path: path.join('bundle-size', commitHash),
-  });
-
   const octokit = new Octokit({
     auth: `token ${process.env.GITHUB_ARTIFACTS_RW_TOKEN}`,
   });
 
-  return octokit.repos
-    .getContents(githubApiCallOptions)
-    .then(() => {
-      log(
-        'The file',
-        cyan(`bundle-size/${commitHash}`),
-        'already exists in the',
-        'build artifacts repository on GitHub. Skipping...'
-      );
-    })
-    .catch(() => {
-      return octokit.repos
-        .createOrUpdateFile(
-          Object.assign(githubApiCallOptions, {
-            message: `bundle-size: ${commitHash} (${bundleSize})`,
-            content: Buffer.from(bundleSize).toString('base64'),
-          })
-        )
+  const promises = [];
+  for (const [compression, extension, getBundleSize] of [
+    ['gzip', '', getGzippedBundleSize],
+    ['brotli', '.br', getBrotliBundleSize],
+  ]) {
+    const bundleSize = `${getBundleSize()}KB`;
+    const commitHash = `${gitCommitHash()}${extension}`;
+    const githubApiCallOptions = Object.assign(buildArtifactsRepoOptions, {
+      path: path.join('bundle-size', commitHash),
+    });
+
+    promises.push(
+      octokit.repos
+        .getContents(githubApiCallOptions)
         .then(() => {
           log(
-            'Stored the new bundle size of',
-            cyan(bundleSize),
-            'in the artifacts',
-            'repository on GitHub'
+            'The file',
+            cyan(`bundle-size/${commitHash}`),
+            'already exists in the',
+            'build artifacts repository on GitHub. Skipping...'
           );
         })
-        .catch(error => {
-          log(
-            red(
-              `ERROR: Failed to create the bundle-size/${commitHash} file in`
-            ),
-            red('the build artifacts repository on GitHub!')
-          );
-          log(red('Error message was:'), error.message);
-          process.exitCode = 1;
-        });
-    });
+        .catch(() => {
+          return octokit.repos
```

Comment:
nit: is it a quick change to unnest these by using await/catch? a bit easier to follow

Reply:
Done. PTAL!

Reply:
LG!